packages/store-engine/README.md

```diff
@@ -18,7 +18,9 @@ Provider for the following storage engines: File, IndexedDB, Memory & LocalStora
 
 Nowadays there are more and more storage possibilities and developers must be familiar with the characteristics of each individual solution to reliably store data. Because it can be sometimes hard to keep up with the highly dynamic world of data storages, we have developed a system which unifies the usage of [IndexedDB](https://developer.mozilla.org/docs/IndexedDB), [In-memory storage](https://en.wikipedia.org/wiki/In-memory_database), [File-based storage](https://nodejs.org/api/fs.html) and [LocalStorage](https://developer.mozilla.org/docs/Web/API/Window/localStorage). In addition we built some functionality (like a transient store which deletes data after a [TTL](https://en.wikipedia.org/wiki/Time_to_live)) on top.
 
-### Overview
+### Quickstart
+
+#### Engines
 
 <!-- prettier-ignore -->
```

Comment:
why ignore?

Reply:
Because Prettier ruins my table layout.

Reply:
How ruined does this look?

```markdown
| Engine             | Available in Browser | Available in Node.js | Description                                                                                                                |
| :----------------- | :------------------: | :------------------: | :------------------------------------------------------------------------------------------------------------------------- |
| FileEngine         |          ❌          |          ✔           | Rudimentary persistent store based on files. Very generic and easy to read.                                                |
| IndexedDBEngine    |          ✔           |          ❌          | Persisent storage which handles significant amounts of structured data, including files/blobs. Enables very fast searches. |
| MemoryEngine       |          ✔           |          ✔           | Transient store which loses data on application restart. Suitable for testing environments.                                |
| LocalStorageEngine |          ✔           |          ❌          | Can save very small amount of data. Stored data is saved across browser sessions. Suitable for simple objects and strings. |
```

Reply:
Exactly... I don't want to have all of these spaces...

I prefer:

```markdown
Engine | Available in Browser | Available in Node.js | Description
:---|:---:|:---:|:---
FileEngine | ❌ | ✔ | Rudimentary persistent store based on files. Very generic and easy to read.
IndexedDBEngine | ✔ | ❌ | Persisent storage which handles significant amounts of structured data, including files/blobs. Enables very fast searches.
MemoryEngine | ✔ | ✔ | Transient store which loses data on application restart. Suitable for testing environments.
LocalStorageEngine | ✔ | ❌ | Can save very small amount of data. Stored data is saved across browser sessions. Suitable for simple objects and strings.
```

Reply:
The whole idea with Prettier is to not care about those! Let's vote: @wireapp/web above.

Reply:
I prefer @lips' version.

Reply:
I just had a random thought: https://twitter.com/apfelbenny/status/966341618092605440

But I will remove `prettier-ignore` to end this discussion here. ;)